src/components/AttachmentModal.js

```diff
@@ -155,6 +155,9 @@ class AttachmentModal extends PureComponent {
      */
     closeConfirmModal() {
         this.setState({isAttachmentInvalid: false});
+        if (_.isFunction(this.props.onModalHide)) {
```

Comment:
I think we should keep this consistent with the similar check above. This prop is also guaranteed to always be passed to the component so I'm not really sure why we have these checks.

```suggestion
        if (this.props.onModalHide) {
```

Reply:
Done.